feature-configs/5g-ref-nw/README.md

```diff
@@ -0,0 +1,222 @@
+
+# __5G RAN reference structure__
+
+# Overview
+This folder contains the manifests for deploying the 5G RAN components on Kubernetes clusters. It complements the [`cn-ran-overlays`](../cn-ran-overlays), that defines a single DU profile, by addressing the manifests scalability aspects. The structure defined here can be extended beyond separate units, approaching closer to the structure maintained by a "typical" mobile network operator.
```

Comment:
What I had in mind is to deprecate cn-ran-overlays eventually, after the new folder can be connected to the CI. Still need to figure out how to handle the transition

> 1/ Do we disable NTP for the DU?

Yes, there is a "disable-chronyd" patch in the current profile that I missed to take in. Fixing that. 
@SchSeba, is there a plan to disable NTP from PTP operator?

> 2/ How would we handle the N3000 bitstream flashing as a precondition? Should we add a check? Is that possible?

There is a [PR-464](https://github.com/openshift-kni/cnf-features-deploy/pull/464/) adding N3000 operator deoplyment, will add the relevant changes to this PR too. 
Regarding the bitstream, I understand that we can't add it since it's proprietary. 
I think we should ask Intel whether they can release a freely open test bitstream, so anyone could load it just to test N3000 hardware is operational. @SchSeba, do you think they have something like that?

Reply:
Added NTP disabling for DUs